modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with the condition type, informing you of the current state of your operator's health. The following definitions cover a list of some common ClusterOperator condition types. Some operators may have additional condition types with operator-specific language, which are not included here. 
+
+Cluster version operator is responsible for collecting these status conditions from cluster operators and represents these is an way which will be easier for OCP administrators to understand the state of the OCP cluster.
+
+//Condition types, as well as additional information about your operator, can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+//[source,terminal]
+//----
+//$ oc get clusterversion -o yaml
+//----
+
+
+* Available: 
+An operator with the condition type `Available` is functional and available in the cluster. If the status is `False`, at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+An operator with the condition type `Progressing` is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. 
++
+Operators do not report as `Progressing` as `True` when they are reconciling a previous known state. If the observed cluster state has changed and the operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
```

Comment:
I'm not sure how to rephrase this to avoid the "as _ as _", but here's one idea:
```suggestion
Operators do not report the condition type `Progressing` as `True` when they are reconciling a previous known state. If the observed cluster state has changed and the Operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
```